src/Core/Silk.NET.Core/Native/Structs/ID3D10Blob.gen.cs

```diff
@@ -22,6 +22,8 @@ namespace Silk.NET.Core.Native
     [NativeName("Name", "ID3D10Blob")]
     public unsafe partial struct ID3D10Blob
     {
+        public static readonly Guid Guid = new("8ba5fb08-5195-40e2-ac58-0d989c3a0102");
```

Comment:
this is defined behaviour right? like there is a spec thing or something? BuildTools isn't just making up GUIDs here?

Reply:
Each interface has an IID defined in the headers which is where we get the GUID.